From 64dbbee7f6570949edc8bca30e6cd026a9e70a59 Mon Sep 17 00:00:00 2001
From: Daniel Stone <daniels@collabora.com>
Date: Fri, 20 Jul 2018 19:00:06 +0100
Subject: [PATCH 17/46] compositor-drm: Add backend pointer to drm_output

Add this for convenience, so it's easier to access when we add the DRM
backend debug scope.

Signed-off-by: Daniel Stone <daniels@collabora.com>
---
 libweston/compositor-drm.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/libweston/compositor-drm.c b/libweston/compositor-drm.c
index 94d78752..612e6a8f 100644
--- a/libweston/compositor-drm.c
+++ b/libweston/compositor-drm.c
@@ -463,6 +463,7 @@ struct drm_head {
 
 struct drm_output {
 	struct weston_output base;
+	struct drm_backend *backend;
 
 	uint32_t crtc_id; /* object ID to pass to DRM functions */
 	int pipe; /* index of CRTC in resource array / bitmasks */
@@ -6104,6 +6105,8 @@ drm_output_create(struct weston_compositor *compositor, const char *name)
 	if (output == NULL)
 		return NULL;
 
+	output->backend = b;
+
 	weston_output_init(&output->base, compositor, name);
 
 	output->base.enable = drm_output_enable;
-- 
2.16.2

